packages/app/src/elements/create-item-dialog.ts

```diff
@@ -139,7 +139,7 @@ export class CreateItemDialog extends Dialog<Vault, VaultItem> {
                     {
                         name: $l("URL"),
                         type: FieldType.Url,
-                        value: parsedUrl.origin + parsedUrl.pathname,
+                        value: parsedUrl.origin,
```

Comment:
Using the full path was giving me a lot of unwieldy urls for the URL field, so I figured it's better to leave it off by default.